dealership_auth/resolver.ts

```diff
@@ -86,6 +86,10 @@ export class AuthResolver {
               field: "username",
               message: "username already taken",
             },
+            {
+              field: "email",
```

Comment:
So this is all that's needed to enforce the unique email? That's cool.

Reply:
this won't enforce it. the uniqueness is enforced in the entity.

Reply:
So this is just the proper error if it's the email that's a duplicate. got it.